packages/roosterjs-editor-core/lib/coreApi/getPendableFormatState.ts

```diff
@@ -67,10 +66,9 @@ const CssFalsyCheckers: Record<PendableFormatNames, (style: CSSStyleDeclaration)
 
 function queryCommandStateFromDOM(
     core: EditorCore,
-    isCollapsed: boolean,
     currentPosition: NodePosition
 ): PendableFormatState {
-    let node = isCollapsed ? currentPosition?.node : document.getSelection().focusNode;
+    let node = currentPosition?.node;
     let formatState: PendableFormatState = {};
     let pendablekeys: PendableFormatNames[] = [];
     while (contains(core.contentDiv, node)) {
```

Comment:
As I comment above, when currentPosition is null, node is also null, so this line will throw exception